src/Components/Counter.js

```diff
@@ -0,0 +1,26 @@
+function Counter({ emoji, fruitTitle, substractProduct, fruitName, addProduct, count }) {
+
+    return (
+        <div className="card">
+            <h2>{emoji} {fruitTitle}</h2>
+            <button
```

Comment:
Ik zou het `disabled` attribuut op `true` zetten als `count` 0 is, want anders kun je een negatief aantal fruit krijgen